src/main/java/prefs/UserConfig.java

```diff
@@ -0,0 +1,7 @@
+package prefs;
+
+/**
+ * Abstraction for user-defined settings.
+ */
+public class UserConfig {
+}
```

Comment:
Can keep it. Change the header comment to `Represents user-defined settings`. The the word `Abstraction` mislead me into think you are trying to define a common parent/interface over settings.